argo-controllers/values.yaml

```diff
@@ -0,0 +1,32 @@
+argoWorkflow:
```

Comment:
should be argoWorkflows

Reply:
enabled field should be added to control whether argoWorkflows should be installed or not